runtime/obj.c

```diff
@@ -667,8 +667,8 @@ rsRetVal objDeserializeProperty(var_t *pProp, strm_t *pStrm)
 finalize_it:
 	if(Debug && iRet != RS_RET_OK && iRet != RS_RET_NO_PROPLINE) {
 		strm.GetCurrOffset(pStrm, &offs);
-		dbgprintf("error %d deserializing property name, offset %lld, step %d\n",
-			  iRet, offs, step);
+		dbgprintf("error %d deserializing property name, offset %llu, step %d\n",
+			  iRet, (long long unsigned)offs, step);
```

Comment:
This consistent with other locations in the code emitting 64 bit unsigned integers.